### ISSUE TITLE
Unresolved reference

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#297]
```
google-api-python-client/samples/adsense/generate_report.py

Unresolved reference to 'gflags'


```

Original issue reported on code.google.com by `D.V.Selitsky@gmail.com` on 15 Aug 2013 at 3:46

**Comment by GoogleCodeExporter** [comment]
```
the same in get_all_custom_channels_for_ad_unit.py
```

Original comment by `D.V.Selitsky@gmail.com` on 15 Aug 2013 at 3:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `jcgregorio@google.com` on 15 Aug 2013 at 3:52
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
please review the whole report (attached)
```

Original comment by `D.V.Selitsky@gmail.com` on 15 Aug 2013 at 4:16

Attachments:
- [code_inspection_rev.7a45e6ed72eb.zip](https://storage.googleapis.com/google-code-attachments/google-api-python-client/issue-297/comment-3/code_inspection_rev.7a45e6ed72eb.zip)

**Comment by GoogleCodeExporter** [comment]
Original comment by `jalc@google.com` on 20 Aug 2013 at 11:17

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 30b30e844a03.
```

Original comment by `jalc@google.com` on 20 Aug 2013 at 1:27
- Changed state: **Fixed**
